Documentation/ceph-cluster-crd.md

```diff
@@ -208,6 +207,28 @@ Some modules will have special configuration to ensure the module is fully funct
 
 * `pg_autoscaler`: Rook will configure all new pools with PG autoscaling by setting: `osd_pool_default_pg_autoscale_mode = on`
 
+### Network Configuration Settings
+
+Configure the network that will be enabled for the cluster and services. This is optional and if not defined then the default SDN will be used.
+
+* `provider`: Specifies the network provider that will be used to connect the network interface. You can choose between `host`, and `multus`.
```

Comment:
What if not specified? You get the default sdn?

Reply:
Yes.

Reply:
How about if the comment above about the default be moved to this line?

Reply:
Need a comment about multus being expermental